templates/commands/render/render.go

```diff
@@ -153,12 +152,12 @@ func (c *Command) realRun(ctx context.Context, rp *runParams) (outErr error) {
 		outErr = errors.Join(outErr, err)
 	}()
 
-	_, templateDir, err := c.downloadTemplate(ctx, rp)
+	_, templateDir, err := templatesource.DownloadTemplate(ctx, rp.fs, rp.tempDirBase, c.flags.Source, c.flags.GitProtocol)
```

Comment:
Maybe we can remove "Template" from the function name since it's already in the package name. https://google.github.io/styleguide/go/decisions#package-vs-exported-symbol-name

Reply:
changed function name to `Download`